### DISCUSSION TITLE
How to use `from` in `CH.Code` tag?

### DISCUSSION THREAD

**Discussion by TangSirOnGit** [code-hike/codehike#336] [General]
Hello,  I want to import a jsx file in `CH.Code` tag like this(then i can config the height or lines of code area), but it doesn't work.
```
<CH.Code>
```js my-button.js 

// from ./my-button.js

```
</CH.Code>

the result is like this :

```
// from ./my-button.js
```

thanks!

```

**Comment by TangSirOnGit** [comment]
sorry for the code format, mycode :

<CH.Code>
```js my-button.js 

// from ./my-button.js

```
</CH.Code>
```

**Reply by pomber** [reply]
Hi, I'll need more details to be able to help

**Reply by TangSirOnGit** [reply]
thank you for replying. I fixed it,:
````
<CH.Code lineNumbers={true} rows={2}>
```js my-button.js 
// from ./my-button.js
```
</CH.Code>
````
